### PR TITLE
Bumped NET Core target frameworks of Test projects to 3.0

### PR DESCRIPTION
**Pull request by FrederikBolding**
Bumped the NET Core target frameworks of the test projects to 3.0

As according to https://dotnet.microsoft.com/download/dotnet-core they've reached end of life.

### CODE REVIEW

**Comment by FrederikBolding** [comment]
@LordMike not sure if we want to bump the version on the main library as well?

**Comment by LordMike** [comment]
We should target netstandard3.0 in addition to the rest of the TFM's.

**Comment by FrederikBolding** [comment]
Do you mean netstandard2.1?

https://docs.microsoft.com/en-us/dotnet/standard/net-standard

**Comment by LordMike** [comment]
Yes. Apparently I do.. explains why my project failed so hard last night :D

File: TMDbLibTests/TMDbLibTests.csproj [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <AssemblyName>TMDbLibTests</AssemblyName>
     <PackageId>TMDbLibTests</PackageId>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
-    <RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>3.0.0</RuntimeFrameworkVersion>
```
Comment (LordMike) [inline TMDbLibTests/TMDbLibTests.csproj]: Remove this entirely.

File: TestApplication/TestApplication.csproj [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <AssemblyName>TestApplication</AssemblyName>
     <OutputType>Exe</OutputType>
     <PackageId>TestApplication</PackageId>
-    <RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>
-    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <RuntimeFrameworkVersion>3.0.0</RuntimeFrameworkVersion>
```
Comment (LordMike) [inline TestApplication/TestApplication.csproj]: Remove this as well.

File: TMDbLibTests/TMDbLibTests.csproj (hunk 1 above)
Comment (FrederikBolding) [reply TMDbLibTests/TMDbLibTests.csproj]: Done!

File: TestApplication/TestApplication.csproj (hunk 2 above)
Comment (FrederikBolding) [reply TestApplication/TestApplication.csproj]: Done!

**Pull request merged**